memecity.game/Menu/Menu.cpp

```diff
@@ -0,0 +1,119 @@
+﻿#include "Menu.h"
```

Comment:
Kan je ervoor zorgen dat `input_manager` hier niet meer gebruikt wordt? Ik zou graag de state de input handling laten doen, en menu's hebben functies met `next` of `previous`